BMM.Core/Translation/en/main.json

```diff
@@ -332,6 +338,22 @@
     "DiscardChanges": "Discard",
     "KeepEditing": "Keep Editing"
   },
+  "ShareTrackCollectionViewModel": {
+    "Title": "Share Playlist",
+    "Done": "Done",
+    "ShareNote": "By sharing a playlist, people will be able to access and listen to the tracks you add, if they have the link.",
+    "ShareLink": "Share link",
+    "MakePrivate": "Make private",
+    "Private": "Private",
+    "SharedWithFormat": "Shared with {0} people",
+    "By": "by"
```

Comment:
There's a difference in letters size. I can change it from code but I'd rather not do it because im not sure if this won't brake different languages grammatical rules.